client/jetpack-cloud/sections/partner-portal/utils.ts

```diff
@@ -176,3 +182,13 @@ export function formatApiPartner( partner: APIPartner ): Partner {
 export function getProductTitle( product: string ): string {
 	return product.replace( /(?:Jetpack\s|[)(])/gi, '' );
 }
+
+export function selectProductOptions( families: APIProductFamily[] ): APIProductFamilyProduct[] {
+	return families.flatMap( ( family ) => family.products );
+}
+
+export function alphabeticallySortedProductOptions(
+	families: APIProductFamily[]
+): APIProductFamilyProduct[] {
+	return sortBy( selectProductOptions( families ), ( product ) => product.name );
+}
```

Comment:
Auxiliary functions for license issuing have been moved from the component level to this utils module.